packages/account/src/Sections/Profile/PhoneVerification/didnt-get-the-code-modal.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import { Button, Modal, Text } from '@deriv-com/quill-ui';
+import { Localize } from '@deriv/translations';
+import { observer, useStore } from '@deriv/stores';
+import { VERIFICATION_SERVICES } from '@deriv/shared';
+import { useGetPhoneNumberOTP } from '@deriv/hooks';
+import { convertPhoneTypeDisplay } from 'Helpers/utils';
+
+type TDidntGetTheCodeModal = {
+    should_show_didnt_get_the_code_modal: boolean;
+    setShouldShowDidntGetTheCodeModal: (value: boolean) => void;
+    setStartTimer: (value: boolean) => void;
+    phone_verification_type: string;
+    setOtpVerification: (value: { show: boolean; phone_verification_type: string }) => void;
```

Comment:
can we have a better variable name for "show" name. i am bit confused what does setOtpVerification ->show variable do?